### ISSUE TITLE
A new Web API Wanted

### ISSUE THREAD

**Issue by 1180300601** [openTCS/opentcs#117]
I hope to add a web api, which can set the location of the vehicle according to the vehicle name:
[POST] /vehicles/{NAME}

NAME(required): The name of the vehicle to be created.
newValue: The position name of the vehicle started.

The Web API function is the same as the Vehicle driver window in the open-TCSKernelControlCenter, whick can set the loopback options of vehicle position. However my java is so poor that i cannot make the API. Plz give me a hand.

**Comment by 1180300601** [comment]
Maybe I am wrong about what i am saying above:

the API should be
[POST] /vehicles/{NAME}/commAdapter/position

NAME(required): The name of the vehicle to be created.
newValue: The position name of the vehicle started.

**Comment by 1180300601** [comment]
![image](https://github.com/openTCS/opentcs/assets/58320555/980d1066-eb8b-4f39-92a1-de857b3e672c)
with this API can only make a web app full

**Comment by swltr** [comment]
@1180300601 I agree that it may be useful to have an endpoint for that in the web API. After all, the Java API also provides a method for that (via `VehicleService.sendCommAdapterCommand()` and an instance of `InitPositionCommand`). So we'd basically need a way to send adapter commands via the web API. No promises about when that will be implemented, though.

In the meantime: You can automatically initialize a virtual vehicle on a specific point in the model. See [the User's Guide](https://opentcs.org/docs/5.12/user/opentcs-users-guide.html#_configuring_a_virtual_vehicles_characteristics) about this. Maybe this is an alternative for you?
